airbyte-integrations/connectors/destination-snowflake/src/main/resources/spec.json

```diff
@@ -19,6 +19,7 @@
         ],
         "type": "string",
         "title": "Host",
+        "pattern": "^([^./?#]+\\.)?([^./?#]+\\.)?([^./?#]+\\.snowflakecomputing\\.com)$",
```

Comment:
Would like to add an `example` https://github.com/airbytehq/airbyte/blob/6f98a68e28c7847091229c2231a61a4bc4abe218/airbyte-integrations/connectors/source-amazon-seller-partner/integration_tests/spec.json#L66 so it's clear when someone types in the wrong host they have an idea of what the code is validating against